### ISSUE TITLE
Add type checking as part of tests

### ISSUE THREAD

**Issue by thiskappaisgrey** [uwsampl/churchroad#53]
Recently, a commit broke something (see: #51 ) broke something that could've been caught if the tests also ran typechecking.

Please also run typechecking as part of tests - to make sure that new commits don't break anything. Thanks.

**Comment by gussmith23** [comment]
Agree, this would be very helpful! Will push it on the queue, but no promises on when it will get done. Feel free to put up a PR!
